chipsec/hal/mmio.py

```diff
@@ -352,18 +352,12 @@ def list_MMIO_BARs(self):
         for _bar_name in self.cs.Cfg.MMIO_BARS:
             if not self.is_MMIO_BAR_defined( _bar_name ): continue
             _bar = self.cs.Cfg.MMIO_BARS[_bar_name]
-            bus_data = None
+            bus_data = []
             if 'register' in _bar:
                 bus_data = self.cs.get_register_bus(_bar['register'])
-                if bus_data is None:
+                if not bus_data:
                     if 'bus' in self.cs.get_register_def(_bar['register']):
                         bus_data = [int(self.cs.get_register_def(_bar['register'])['bus'],16)]
-                    else:
-                        # On AMD, some BARs are defined in MSRs or SMNs. These are associated
-                        # with a cpu thread or a die (respectively), which can be mapped to a
-                        # northbridge bus, but we don't support it currently. Just pass a default
-                        # bus, and let downstream code fallback on die 0.
-                        bus_data = [None]
```

Comment:
Does AMD look for `[None]`? Because `[]` != `[None]`

Reply:
Unknown.  This is the only instance of this I could find.